### PR TITLE
docs(react-query): add docs for usePrefetchInfiniteQuery

### PR DESCRIPTION
**Pull request by manudeli**
# Overview

I added docs for usePrefetchInfiniteQuery and I added @gwansikk @kangju2000 as co-author

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: docs/suspensive.org/src/pages/docs/react-query/usePrefetchQuery.en.mdx [hunk 1]
```diff
@@ -5,10 +5,13 @@ import { Callout, Sandpack } from '@/components'
 The usePrefetchQuery does not return anything, it should be used just to fire a prefetch during render, before a suspense boundary that wraps a component that uses useSuspenseQuery.
 
 ```jsx /usePrefetchQuery/
-import { usePrefetchQuery, queryOptions } from '@suspensive/react-query'
```
Comment (manudeli) [inline docs/suspensive.org/src/pages/docs/react-query/usePrefetchQuery.en.mdx:8]: I removed queryOptions that can make readers confused by unnecessary context

File: docs/suspensive.org/src/pages/docs/react-query/usePrefetchQuery.en.mdx [hunk 2]
```diff
@@ -64,14 +58,12 @@ export const Example = () => {
     </div>
   )
 }
-```
 
-```tsx Post.tsx
```
Comment (manudeli) [inline docs/suspensive.org/src/pages/docs/react-query/usePrefetchQuery.en.mdx:69]: No depth to more Intuition

File: docs/suspensive.org/src/pages/docs/react-query/usePrefetchQuery.en.mdx [hunk 3]
```diff
@@ -80,28 +72,11 @@ export const Post = ({ postId }: { postId: number }) => {
...
   )
 }
-```
 
-```tsx /query.ts
-import { queryOptions } from '@suspensive/react-query'
-
-export const query = {
-  post: (postId: number) =>
-    queryOptions({
```
Comment (manudeli) [inline docs/suspensive.org/src/pages/docs/react-query/usePrefetchQuery.en.mdx:90]: no queryOptions for easy understanding

**Review by gwansikk — APPROVED** [review]
ready for suspensive! 👍👍👍

**Pull request merged**